init.rb

```diff
@@ -43,11 +43,31 @@
     }
 end
 
+module RedmineEquipmentStatusViewer
+  module ControllerHelper
+    def is_mobile?
+      request.user_agent =~ /Mobile|Blackberry|Android/
+    end
+
+    def mobile_device?
+      if session[:mobile_param]
+        session[:mobile_param] == "1"
+      else
+        is_mobile?
+      end
+    end
+
+    def save_mobile_param
+      unless params[:mobile].blank?
+        session[:mobile_param] = params[:mobile]
+      end
+    end
+  end
+end
+
 Redmine::Search.register :equipment_assets
 
-# Found redmine-1.4 -> redmine-2.1 porting info at http://www.redmine.org/boards/3/topics/30423
-#require 'dispatcher'
 Rails.configuration.to_prepare do
   require_dependency 'search_controller'
-  SearchController.send(:include, RedmineEquipmentStatusViewer::Patches::SearchControllerPatch)
+  # SearchController.send(:include, RedmineEquipmentStatusViewer::Patches::SearchControllerPatch)
```

Comment:
@phlegx can this be merge with the patch or is Redmine now doing that automatically? What is the effect of commenting this line out? Once answered I can merge.